From a3ad3b3e828191f0720b71058cffa7b6678e421a Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 1 Oct 2024 18:03:33 +0200
Subject: [PATCH 1263/1695] dts: stm32: add wakeup properties to the RTC for
 stm32mp23 plateform

Add 'wakeup-source' property in stm32mp231.dtsi

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 844de6520fe389fd03766d78b62f28eea3b0414f dts: stm32: add wakeup properties to the RTC for stm32mp23 plateform]
cf: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/402363
Change-Id: I536c1ff30b840b4859b56abff36f3fa9371ce285
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/408147
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 02dafae78..4eb724a58 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -3123,6 +3123,8 @@
 			reg = <0x46000000 0x400>;
 			clocks = <&rcc CK_BUS_RTC>, <&rcc RTC_CK>;
 			clock-names = "pclk", "rtc_ck";
+			wakeup-source;
+			wakeup-parent = <&exti2>;
 			interrupts = <GIC_SPI 15 IRQ_TYPE_LEVEL_HIGH>;
 		};
 
-- 
2.47.2

